### PR TITLE
Update Cost-Burdened Renters metric to 2019

### PR DESCRIPTION
**Pull request by Vanie2020**
2019 ACS metrics and world values

### CODE REVIEW

File: metric/blockgroup/dUNFOWN.csv [hunk 1]
```diff
@@ -1,154 +1,154 @@
...
-370630020282,0,16,0,0,10,14,85,133
-370630020283,155,228,261,362,373,465,535,634
-370630021001,473,431,470,494,478,517,609,605
-370630021002,546,569,604,628,643,652,639,644
-370630021003,452,476,456,464,475,443,442,375
-370630022001,132,146,148,168,151,175,175,182
-370630023001,11,12,23,7,7,13,20,41
-370630023002,0,7,4,1,4,4,0,0
-370639801001,25,39,32,25,0,0,0,0
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
```
Comment (JohnKilleen) [inline metric/blockgroup/dUNFOWN.csv:1]: @BiSeye This one is replacing the dUNFOWN table with the rUNFOWN table data. Be careful to write the dUNFOWN csv from the dUNFOWN table in R. To check it here I compared the first row with the Compass ([https://compass.durhamnc.gov/en/compass/UNFOWN/blockgroup?selected=370630001011]) which shows 37, 48, 43, 34 households in the numerator each recent year.

File: metric/blockgroup/mUNFRENT-accuracy.csv [hunk 2]
```diff
@@ -1,154 +1,154 @@
...
-370630020283,11.18186055,9.337881552,10.50710532,11.97221396,14.49932007,11.46461158,12.0897362403739,11.9009744030715
-370630021001,188.5081406,38.32222446,45.44556941,27.16942443,49.26254239,45.89392237,67.3796676094251,38.3786718850158
-370630021002,83.46616694,40.84666743,53.56735474,78.67714853,76.72674832,52.10144984,48.0758475615562,40.5978949971043
-370630021003,106.6548708,31.85961974,35.52942223,65.87909797,74.80962807,72.18162273,77.1357116838583,40.6058778769512
-370630022001,22.27882011,12.99322133,10.84015761,11.09328707,9.296149917,9.11298339,10.3652860775217,11.8732357101498
-370630023001,18.07703497,20.73347089,22.81152461,21.10592019,17.53896792,20.62808672,32.5942919376448,22.2804421934026
-370630023002,319.2184217,40.53893441,20.39576453,32.08990142,31.05912285,24.10559948,26.5271550210798,15.1925633349006
-370639801001,,,,,,,,
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
+370630001011,56.20680272,26.57041309,24.28510594,28.47672003,26.10530482,26.93677964,27.2842631665674,27.2546690037882,53.1839614057274
```
Comment (JohnKilleen) [inline metric/blockgroup/mUNFRENT-accuracy.csv:2]: @BiSeye The new values here are consistently much higher than prior years. This makes me wonder if the numerator (estimate) used for the calculation might have been one of the renter categories but not all? For example renters paying 30-34% of their income on housing costs (B25070_007), but not the other categories (B25070_008, B25070_009, etc).

File: metric/blockgroup/dUNFOWN.csv (hunk 1 above)
Comment (Vanie2020) [reply metric/blockgroup/dUNFOWN.csv:1]: I repushed the correction.  Yes i caught the mistake in my script

Comment (timstallmann) [reply metric/blockgroup/dUNFOWN.csv:1]: @BiSeye did you repush to this branch?

**Comment by Vanie2020** [comment]
Tim, what else in needed here so im clear about this assignment to me

**Comment by timstallmann** [comment]
Think it's mostly the `dUNFOWN.csv` table for blockgroups which seems to be off

**Comment by Vanie2020** [comment]
Thanks Tim! Should be good to go now.

**Comment by timstallmann** [comment]
Looks good to me now!

**Pull request merged**